test/mesh_test.cpp

```diff
@@ -145,20 +146,20 @@ TEST(Manifold, GetMesh) {
   Identical(mesh_out, mesh_out2);
 }
 
-TEST(Manifold, Regression) {
-  Manifold manifold(ImportMesh("data/gyroidpuzzle.ply"));
-  EXPECT_TRUE(manifold.IsManifold());
-
-  Manifold manifold1 = manifold;
-  manifold1.Translate(glm::vec3(5.0f));
-  int num_overlaps = manifold.NumOverlaps(manifold1);
-  ASSERT_EQ(num_overlaps, 228677);
-
-  Mesh mesh_out = manifold.GetMesh();
-  Manifold manifold2(mesh_out);
-  Mesh mesh_out2 = manifold2.GetMesh();
-  // Identical(mesh_out, mesh_out2);
-}
+//TEST(Manifold, Regression) {
+//  Manifold manifold(ImportMesh("c:/repos/manifold/test/data/gyroidpuzzle.ply"));
```

Comment:
Someone must have found way to deal with this stupid Linux vs Windows file system stuff by now, right? I think we'll want to stick with relative paths too. I don't want to merge this until we find a way to uncomment these tests.

Reply:
Relative paths like "./foo" should work just fine in windows.

But you'll need to consider the environment you're invoking the test from.

Try printing std::filesystem::current_path() to see where the paths should be relative to.